external/RSL_LITE/rsl_bcast.c

```diff
@@ -80,6 +80,26 @@ static destroy_par_info ( p )
   if ( p != NULL ) RSL_FREE( p ) ;
 }
 
+static destroy_list( list, dfcn )
+     rsl_list_t ** list ;          /* pointer to pointer to list */
+     int (*dfcn)() ;               /* pointer to function for destroying
+                                      the data field of the list */
+{
+  rsl_list_t *p, *trash ;
+  if ( list == NULL ) return(0) ;
+  if ( *list == NULL ) return(0) ;
+  for ( p = *list ; p != NULL ; )
+    {
```

Comment:
It's a minor issue, but is there a reason for changing the indentation of the for-loop?

Reply:
No, this was the auto-formatter. I will change it back.